### ISSUE TITLE
Panic in CommitBatcher

### ISSUE THREAD

**Issue by RobQuistNL** [filecoin-project/lotus#9377]
### Checklist

- [X] This is **not** a security-related bug/issue. If it is, please follow please follow the [security policy](https://github.com/filecoin-project/lotus/security/policy).
- [X] This is **not** a question or a support request. If you have any lotus related questions, please ask in the [lotus forum](https://github.com/filecoin-project/lotus/discussions).
- [X] This is **not** a new feature request. If it is, please file a [feature request](https://github.com/filecoin-project/lotus/issues/new?assignees=&labels=need%2Ftriage%2Ckind%2Ffeature&template=feature_request.yml) instead.
- [X] This is **not** an enhancement request. If it is, please file a [improvement suggestion](https://github.com/filecoin-project/lotus/issues/new?assignees=&labels=need%2Ftriage%2Ckind%2Fenhancement&template=enhancement.yml) instead.
- [X] I **have** searched on the [issue tracker](https://github.com/filecoin-project/lotus/issues) and the [lotus forum](https://github.com/filecoin-project/lotus/discussions), and there is no existing related issue or discussion.
- [X] I am running the [`Latest release`](https://github.com/filecoin-project/lotus/releases), or the most recent RC(release canadiate) for the upcoming release or the dev branch(master), or have an issue updating to any of these.
- [ ] I did not make any code changes to lotus.

### Lotus component
- [X] lotus miner

### Lotus Version

```text
1.17.0
```

### Describe the Bug

Miner crashed, after restarting it it keeps crashing while doing the wdpost (but it can never finish it);

```
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x70 pc=0x1a502e1]

goroutine 2279810 [running]:
github.com/filecoin-project/lotus/storage/pipeline.(*CommitBatcher).getCommitCutoff(_, {{0xc08397a348, 0x15}, 0x83cc, 0x8, 0x0, {0xc08fd332c0, 0x1, 0x1}, {0xc099df2820, ...}, ...})
	/home/dukesoft/lotus/storage/pipeline/commit_batch.go:587 +0x3a1
github.com/filecoin-project/lotus/storage/pipeline.(*CommitBatcher).AddCommit(_, {_, _}, {{0xc08397a348, 0x15}, 0x83cc, 0x8, 0x0, {0xc08fd332c0, 0x1, ...}, ...}, ...)
	/home/dukesoft/lotus/storage/pipeline/commit_batch.go:478 +0xa5
github.com/filecoin-project/lotus/storage/pipeline.(*Sealing).handleSubmitCommitAggregate(_, {{_, _}, _}, {{0xc08397a348, 0x15}, 0x83cc, 0x8, 0x0, {0xc08fd332c0, ...}, ...})
	/home/dukesoft/lotus/storage/pipeline/states_sealing.go:691 +0x21b
github.com/filecoin-project/lotus/storage/pipeline.(*Sealing).Plan.func1({{_, _}, _}, {{0xc08397a348, 0x15}, 0x83cc, 0x8, 0x0, {0xc08fd332c0, 0x1, ...}, ...})
	/home/dukesoft/lotus/storage/pipeline/fsm.go:32 +0x76
reflect.Value.call({0x38ce7a0, 0xc06288daa0, 0x13}, {0x3ce03df, 0x4}, {0xc023b28f60, 0x2, 0x2})
	/usr/local/go/src/reflect/value.go:556 +0x845
reflect.Value.Call({0x38ce7a0, 0xc06288daa0, 0xc01b728221}, {0xc031491760, 0x2, 0x2})
	/usr/local/go/src/reflect/value.go:339 +0xc5
github.com/filecoin-project/go-statemachine.(*StateMachine).run.func3()
	/home/dukesoft/go/pkg/mod/github.com/filecoin-project/go-statemachine@v1.0.2/machine.go:113 +0x36a
created by github.com/filecoin-project/go-statemachine.(*StateMachine).run
	/home/dukesoft/go/pkg/mod/github.com/filecoin-project/go-statemachine@v1.0.2/machine.go:109 +0x6bf
```

**Comment by Reiers** [comment]
**Hi @RobQuistNL** 

**Thanks for the report.** 
Could you elaborate a bit more more - and share some more lotus logs from the incident? Thanks!

**Comment by RobQuistNL** [comment]
Might be fixed now that I upgraded to 1.17.1, might be lucky. I don't know.
